packages/aws-cdk-lib/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -60,6 +60,21 @@ describe('', () => {
       expect(pipeline.pipelineName).toEqual('MyPipeline');
     });
 
+    test('imported pipeline has correct region and account', () => {
+      const stack = new cdk.Stack(new cdk.App(), 'Stack', {
+        env: {
+          region: 'eu-west-1',
+          account: '234567890123',
+        },
+      });
+
+      const pipeline = codepipeline.Pipeline.fromPipelineArn(stack, 'Pipeline',
+        'arn:aws:codepipeline:us-east-1:123456789012:MyPipeline');
+
+      expect(pipeline.env.region).toEqual('us-east-1');
+      expect(pipeline.env.account).toEqual('123456789012');
```

Comment:
```suggestion
      const stack = new cdk.Stack();

      const pipeline = codepipeline.Pipeline.fromPipelineArn(stack, 'Pipeline',
        'arn:aws:codepipeline:us-west-1:123456789012:MyPipeline');

      expect(pipeline.env.region).toEqual('us-west-1');
      expect(pipeline.env.account).toEqual('123456789012');
```
Let's keep it simple.

Reply:
I prefer adding the env to the stack explicitly, because it shows that the import really does not use the environment of the stack 🤔